plasmapy/analysis/tests/test_nullpoint.py

```diff
@@ -491,6 +500,7 @@ def test_null_point_find10():
             assert p.classification == "Continuous concentric ellipses"
 
 
+@pytest.mark.slow
```

Comment:
No surprises there, given #1425 is still outstanding!

- [ ] once this is merged, we should edit that one to check if these can be removed.